subsys/bluetooth/host/gatt.c

```diff
@@ -654,34 +771,33 @@ static void sc_indicate_rsp(struct bt_conn *conn,
 
 static void sc_process(struct k_work *work)
 {
-	struct gatt_sc *sc = CONTAINER_OF(work, struct gatt_sc, work);
```

Comment:
It doesn't seems we changed the storage of gatt_sc so I wonder why you are chaning the handling here?

Reply:
@joerchan suggested this change a few hours ago and I agreed!
"gatt_sc is a singleton. Should we maybe remove the CONTAINER_OF in this function, and use gatt_sc?"

Reply:
I prefer the way it was since that would be easier to change if we ever change the gatt_sc works.

Reply:
I'm fine with either ways. @joerchan, @jhedberg, any strong opinions on this before I change it back to CONTAINER_OF? :-D

Reply:
I actually just reverted to use CONTAINER_OF in latest push, it seems that it is common practice with many workQ jobs and it would allow for more flexibility.

Reply:
That's alright with me.